<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UAAReport</class>
 <widget class="QDialog" name="UAAReport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Report user to UAA</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelUsername">
     <property name="text">
      <string>This user's username is:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxDisruptive">
     <property name="text">
      <string>Disruptive</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxOffensive">
     <property name="text">
      <string>Offensive</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxPromotional">
     <property name="text">
      <string>Promotional</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxMisleading">
     <property name="text">
      <string>Misleading</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelOtherReason">
     <property name="text">
      <string>Other reason (optional):</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditOtherReason"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCheck">
       <property name="text">
        <string>Check if user is reported</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnReport">
       <property name="text">
        <string>Report user</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
